test/fixtures/linter/rules/NoDeprecatedApi/ui5.yaml

```diff
@@ -0,0 +1,11 @@
+specVersion: '3.0'
```

Comment:
Please add a test case for a YAML with multiple documents.

For example a project- and custom task configuration in one file as shown here: https://sap.github.io/ui5-tooling/stable/pages/extensibility/CustomTasks/#example-custom-task-extension-defined-in-ui5-project

Reply:
Resolved?